templates/CRM/Admin/Form/Setting/Case.tpl

```diff
@@ -37,6 +37,12 @@
         <span class="description">{ts}Enable embedded tracking to activity revisions within the "civicrm_activity" table. Alternatively, see "Administer => System Settings => Misc => Logging".{/ts}</span>
       </td>
     </tr>
+    <tr class="crm-case-form-block-civicaseShowCaseActivities">
+      <td class="label">{$form.civicaseShowCaseActivities.label}</td>
+      <td>{$form.civicaseShowCaseActivities.html}<br />
+        <span class="description">{ts}e.g. the Contact form's Activity tab listing. Without this ticked, activities that belong to a case are hidden (default behaviour).{/ts}</span>
```

Comment:
Instead of copy pasting text you can rewrite to grab it from settings
`<span class="description">{$descriptions. civicaseShowCaseActivities}</span>`

or 

`<span class="description">{$civicaseShowCaseActivities_description}</span>`